### PR TITLE
New Alt-Titles[Need Feedback] 

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request
Add new alt-titles to several jobs:
I only need feeback for blackshield and Marshal alt-titles:

Troopers: Blackshield Technician
Corpsmen: Trainee Corpsman, Blackshield Medical Specialist , Combat Surgeon
Ranger: Detective , Marshal Agent

### CODE REVIEW

**Comment by TheSleepyCatmom** [comment]
Marshal Agents are their own thing, the High Council sends them.

**Comment by Guidesu** [comment]
> Marshal Agents are their own thing, the High Council sends them.

"The Ranger is the colony detective and field agent, taking on cases and suspects that aren't always what they seem"

File: code/game/jobs/job/prospector.dm [hunk 1]
```diff
@@ -105,6 +106,7 @@
 	spawn_positions = 2
 	supervisors = "the Foreman"
 	difficulty = "Medium."
+	alt_titles = list("Pathfinder","Sapper","Scout","Frontiersmen")
```
Comment (Trilbyspaceclone) [inline code/game/jobs/job/prospector.dm:109]: Not sure if pathfinder is a good idea

Comment (Guidesu) [reply code/game/jobs/job/prospector.dm:109]: Why not?

Comment (Trilbyspaceclone) [reply code/game/jobs/job/prospector.dm:109]: I dont think they are explorers or really SI related

Comment (Guidesu) [reply code/game/jobs/job/prospector.dm:109]: Why so?

Comment (Guidesu) [reply code/game/jobs/job/prospector.dm:109]: I  mean, prospectors open the way for the salvagers, they are frontliners

Comment (r4d6) [reply code/game/jobs/job/prospector.dm]: Prospectors are the muscles, Salvagers are the brains, Foreman's supposed to be both.

**Comment by Guidesu** [comment]
Going make the suggestion later.

**Comment by cdb-is-not-good** [comment]
Corpsman/Ranger were left without alt titles for a reason. Their titles are particular and their duties encompass a wide swath. As for black shield tech the idea of a more technical oriented shield was also shot down- especially for a trooper, that steps on the guilds toes for no good reason

**Pull request merged**